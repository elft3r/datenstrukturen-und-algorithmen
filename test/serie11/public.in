8
3 468 664 291 714 338 218
5 577 524 160 456 384 873 903 858 110 22
10 211 195 312 422 295 47 914 340 976 534 139 976 584 709 784 16 957 64 550 727
15 605 171 917 917 229 570 749 279 164 618 187 926 998 182 564 195 558 717 87 126 709 506 738 381 919 573 418 730 336 31
20 625 684 201 486 108 256 807 391 763 342 812 174 727 714 681 301 50 147 200 429 557 790 518 216 488 473 918 373 324 301 586 58 730 527 953 779 925 408 822 543
50 890 467 356 124 751 351 203 955 346 255 461 343 466 709 945 416 510 383 221 445 318 432 174 971 777 842 453 459 354 158 2 58 143 973 24 385 111 563 147 183 199 70 263 102 658 751 516 850 476 865 135 410 60 935 420 86 379 282 962 666 636 912 313 101 157 79 726 690 583 97 223 932 656 265 240 891 844 101 302 97 813 410 723 801 285 124 115 618 116 240 554 459 558 655 631 940 508 139 261 338
100 229 244 21 597 455 436 11 752 20 342 721 666 142 606 140 937 277 456 390 337 39 774 521 257 992 149 681 160 288 126 787 202 991 603 992 801 650 143 701 149 131 782 599 341 977 748 221 52 602 426 833 475 429 775 21 521 682 394 214 141 920 371 524 971 549 95 703 424 502 217 722 781 983 35 469 773 671 698 315 669 79 703 857 883 808 397 973 702 290 326 437 772 236 544 724 656 421 236 320 890 610 299 791 644 414 850 132 331 615 993 377 225 15 523 320 442 198 721 211 422 681 775 694 420 795 894 877 342 318 647 358 698 292 644 210 346 702 190 458 189 679 212 299 951 674 336 393 712 626 686 884 914 465 668 398 155 455 728 176 185 744 825 672 201 226 166 385 669 237 557 449 677 912 579 501 993 12 690 675 848 885 256 102 714 788 482 211 215 192 154 227 667 849 156 700 642 325 737 605 386
500 430 580 534 873 487 829 768 791 651 377 566 752 547 848 837 594 370 950 474 728 175 60 13 93 533 186 955 436 879 735 63 317 882 835 136 446 899 314 333 706 105 824 879 483 213 999 72 756 795 13 889 681 86 354 103 345 797 134 771 698 931 839 132 652 798 929 468 756 410 367 218 890 263 735 715 912 29 293 202 266 55 27 690 445 119 882 346 661 294 32 919 81 891 508 651 828 595 193 742 908 820 421 110 980 577 630 442 535 682 744 335 213 612 718 657 981 442 991 441 883 725 557 553 141 896 744 98 438 739 775 31 310 747 672 710 740 738 105 210 168 925 305 55 366 751 299 634 836 177 287 4 642 815 192 951 897 206 26 147 429 830 991 17 505 903 939 429 897 861 91 659 952 161 804 811 2 565 340 366 468 968 20 607 189 24 221 659 136 565 319 316 898 490 345 832 317 417 676 867 284 192 884 307 974 662 577 256 408 264 770 446 309 880 107 427 914 109 28 394 543 554 803 220 463 992 215 295 828 332 533 52 227 666 719 427 917 241 20 234 83 965 919 753 16 133 879 853 571 625 210 655 327 440 161 757 888 319 305 455 472 322 650 782 449 215 945 48 992 741 900 542 239 170 151 602 25 346 994 670 659 920 924 309 816 912 643 260 459 237 593 478 982 193 470 109 567 973 623 763 95 133 742 79 795 637 722 476 253 504 127 220 286 327 287 401 698 920 617 182 881 137 409 753 792 469 4 705 853 373 959 209 934 413 520 222 697 183 514 706 532 866 925 439 104 582 867 278 424 402 39 991 63 323 29 887 146 603 463 125 422 58 261 76 891 214 901 43 313 676 804 872 658 327 631 323 769 462 6 665 310 692 502 406 848 600 499 94 103 164 659 32 43 614 117 202 881 729 525 248 79 437 637 153 739 84 240 629 450 747 99 413 524 108 759 586 24 792 617 468 923 946 796 425 318 575 863 465 105 641 372 748 489 841 581 82 23 152 501 77 81 116 770 21 589 313 907 275 760 525 290 97 929 845 263 496 155 488 888 255 481 253 250 761 490 404 391 289 802 63 722 315 962 518 63 845 732 873 27 95 688 762 118 837 971 180 784 341 241 912 353 901 243 793 799 326 679 180 139 173 318 937 797 306 944 473 586 431 207 187 755 770 515 901 666 437 482 54 905 824 450 402 390 296 169 768 117 401 237 243 157 192 364 423 229 596 711 703 253 904 484 228 869 248 293 580 437 569 265 790 381 100 909 611 37 598 954 25 876 724 115 347 195 952 235 79 739 186 374 678 611 283 437 950 266 142 225 942 618 99 76 694 575 381 38 245 427 453 358 140 596 195 428 823 268 182 432 836 336 619 997 467 227 76 277 718 58 599 898 809 876 250 281 428 113 325 269 514 343 26 390 636 422 880 760 952 417 703 826 851 807 108 186 632 962 645 193 23 398 807 528 489 228 509 594 212 532 861 129 897 270 835 337 61 969 944 499 666 812 825 746 255 654 998 328 116 372 838 263 540 832 863 157 568 196 707 916 894 550 500 923 484 302 361 889 807 778 999 44 663 560 454 418 5 487 31 469 998 181 944 18 90 111 420 840 945 764 383 68 716 350 592 629 356 39 301 128 359 356 780 480 575 574 408 527 158 237 18 534 749 123 144 583 916 875 756 730 454 579 258 557 26 645 210 425 357 816 895 699 46 833 916 635 655 568 90 570 502 644 374 701 88 752 685 154 972 684 183 726 394 96 624 598 873 774 1 339 122 563 881 322 849 233 13 928 643 517 143 425 153 750 634 185 847 269 576 588 806 292 610 873 644 781 359 150 379 669 796 838 582 822 954 41 886 586 764 863 932 860 592 803 954 471 311 114 329 418 617 155 617 641 118 471 329 434 628 980 462 787 268 334 89 638 564 34 906 271 222 240 862 623 211 731 399 606 144 185 525 320 73 629 28 315 142 773 568 174 632 724 455 948 805 862 573 224 828 188 198 404 384 356 29 969 364 127 786 385 43 164 891 399 805 933 71 83 457 527 258 272 733 563 176 837 528 668 988 603 102 171 22 220 639 990 545 641 857 297 418 29 705 809 40 427 574 149 181 34 172 933 851 444 399 489 465 813 44 174 997 781 352 663 198 370 67 477 64 762 710 336 776 406 698 24 993 473 130 447 70 669 288 553 344 205 64 348 4 922 262 398 100 169 974 870 929 284 224 258 635 900 315 430 367 319 164 439 877 660 799 697 821 322 475 277 791 1 945 59 259 56 987 496
