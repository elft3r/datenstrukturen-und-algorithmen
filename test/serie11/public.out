291 714 468 664 338 218
110 22 160 456 384 873 903 858
139 976 976 534 957 64 784 16 295 47 211 195
87 126 187 926 917 917 998 182 336 31
50 147 201 486 557 790 953 779 925 408 918 373 812 174 586 58
2 58 60 935 143 973 174 971 631 940 777 842 962 666 945 416 844 101
11 752 140 937 501 993 615 993 884 914 992 801 992 149 983 35 221 52 20 342 15 523 12 690
1 339 4 922 29 969 48 992 213 999 619 997 830 991 965 919 997 781 999 44 968 20 811 2 791 1 469 4 82 23 55 27 32 43 13 93
